include/ceed-impl.h

```diff
@@ -95,10 +97,13 @@ struct CeedBasis_private {
                CeedScalar *);
   int (*Destroy)(CeedBasis);
   int refcount;
+  bool tensorbasis;      /* flag for tensor basis */
   CeedInt dim;           /* topological dimension */
   CeedInt ncomp;         /* number of field components (1 for scalar fields) */
   CeedInt P1d;           /* number of nodes in one dimension */
   CeedInt Q1d;           /* number of quadrature points in one dimension */
+  CeedInt P;             /* total number of nodes */
+  CeedInt Q;             /* total number of quadrature points */
```

Comment:
I added 'P' and 'Q' because there isn't a conventional mapping from 'P1d' and 'Q1d' to total degrees of freedom or total quadrature points for non-tensor bases.